src/routes/controllers/projects/getSuggestions.js

```diff
@@ -53,6 +66,18 @@ const getSuggestions = async (req, res) => {
                     rowMode: 'array',
                 });
                 row.tags = tags.rows.flat();
+
+                const matchingTagCount = await pool.query(
+                    `SELECT COUNT(*) 
+                            FROM "ProjectTag" as pt 
+                            JOIN "UserInterest" as ui 
+                            ON pt.tag = ui.interest 
+                            WHERE project_id = $1 
+                            AND ui.username = $2`,
+                    [row.id, username]
+                );
+                row.tagAffinity = Math.round(matchingTagCount.rows[0].count / row.tags.length * 100);
```

Comment:
[**Urgent**] The computation here will be performed from left to right, correct? So result of `matchingTagCount.rows[0].count / row.tags.length` will be multiplied by `100`?

Reply:
Yeah, it would be the same thing as doing `(matchingTagCount.rows[0].count / row.tags.length) * 100`.